jobs/jobs-operator/manifests/base/job-runner-crd.yaml

```diff
@@ -25,6 +25,9 @@ spec:
                   type: integer
                 seconds_to_poll_db:
                   type: integer
+                product_specific:
+                  type: object
+                  x-kubernetes-preserve-unknown-fields: true
```

Comment:
Does anyone have an opinion if they think I should capitalize all of these env variables in this CRD?

Reply:
What would that look like?